pkg/occlient/occlient.go

```diff
@@ -3241,3 +3241,17 @@ func (c *Client) PropagateDeletes(targetPodName string, delSrcRelPaths []string,
 	}
 	return err
 }
+
+// GetCreateType returns enum equivalent of passed component source type or error if unsupported type passed
+func GetCreateType(ctStr string) (CreateType, error) {
```

Comment:
why this can' be in config pkg? It should be probably called `SourceType` to be in pair what we have in the config

Reply:
I am anyway already moving it as part of my PR I'll do it in #1360 if its ok bcoz altering The CreateType location will have wide impact on this PR and I am anyway touching it in #1360

Reply:
so can this be resolved?

Reply:
Yeah it will be resolved in #1360 
As long as this PR doesn't suffer cyclic import ;)

Reply:
resolving this